### ISSUE TITLE
Drop serie tags while dropping another serie

### ISSUE THREAD

**Issue by bacondc** [influxdata/influxdb#8456]
__Steps to reproduce:__

1. I had a serie (Serie1) on our database defined by one tag (tag1)
2. I injected new datas with same tag as Serie1 and a new tag (tag2).
3. I dropped the serie defined by tag1 and tag2 using DROP SERIES WHERE tag1='valuetag1' and tag2='valuetag2' to drop the points injected at step 2.
4. I queried our database using SELECT * FROM m WHERE tag1='valuetag1' expecting to get the Serie1 as it was at step 1 but the result was a serie where tag1 has 'disapeared' (blank field in the web admin interface and None field throught the python API).

I don't understand what happened there, I supposed the tag is 'still there' because I can still make query on it but I can't see its value.

**Comment by jwilder** [comment]
Can you distill your repro steps into actual `insert` and `selects` that we can run to reproduce the issue?

Also, what version are you using?

**Comment by bacondc** [comment]
I'm using InfluxDB 1.2.0. I tried to build another database and to reproduce the case that I explained previously but no bugs appear : when I use DROP SERIES with WHERE on tags, the series which match the WHERE clause are deleted but the tags of the other series don't "disappeared" as it occured in my original database.

Moreover, I just noticed that all tags of all series have been "cleaned" of my database (except the tags of the series which have been builded when I injected new datas to my original series adding a new tag to these datas to be able to find them easily) : I can't see them when I query with web interface or python api, but I can still query on these tag to find datas of a serie.
